test/CMakeLists.txt

```diff
@@ -0,0 +1,20 @@
+configure_lit_site_cfg(
+        ${CMAKE_CURRENT_SOURCE_DIR}/lit.site.cfg.py.in
+        ${CMAKE_CURRENT_BINARY_DIR}/lit.site.cfg.py
+        MAIN_CONFIG
+        ${CMAKE_CURRENT_SOURCE_DIR}/lit.cfg.py
+)
+
+set(GC_OPT_TEST_DEPENDS
+        FileCheck count not
+        # mlir-gen
+        gc-opt
+        )
+
+add_lit_testsuite(gc-check "Running the regression tests"
+        ${CMAKE_CURRENT_BINARY_DIR}
+        DEPENDS ${GC_OPT_TEST_DEPENDS}
+        )
+set_target_properties(gc-check PROPERTIES FOLDER "Tests")
```

Comment:
What is "Tests" pointing to?

Reply:
In some cases written that it used for IDE. I am not sure what it does, added TODO for this.